### ISSUE TITLE
Duda orden encriptación y decodificación

### ISSUE THREAD

**Issue by MaxAl100** [IIC2233/Syllabus#255]
### Prerrequisitos
(Marcar colocando una X entre los corchetes los ítems que ya hiciste, así: "[X]")

* [x] Leí las reglas del foro (https://github.com/IIC2233/syllabus/issues/4)
* [x] Busqué en las *issues* si ya preguntaron mi duda y no encontré nada parecido (https://github.com/IIC2233/syllabus/issues)
* [x] Revisé el compilado de dudas de la tarea y no encontré una *issue* similar a la mía (https://github.com/IIC2233/syllabus/issues/1)
* [x] Mi duda no se trata sobre una librería, *built-in* o mala práctica, ya que eso se pregunta en la *issue* creada para ello.
* [x] Mi duda no se trata de un tema administrativo o personal, ya que en ese caso debo contactar a mi profe, al Jefe de Bienestar o al correo del curso (https://iic2233.github.io/contacto/)
* [x] Utilizaré un título descriptivo y llenaré correctamente esta plantilla
* [x] De ser necesario, solo colocaré código simple que permita explicar mi problema o duda, ya que compartir código de esta evaluación puede ser considerado copia

### Duda

Sólo para asegurarme, pero primero hay que encriptar la información de cada bloque y luego codificarla en los bloques y lo demás, ¿no? Esto es para asegurarme de que lo esté programando bien :sweat_smile:

**Comment by vicentethomas** [comment]
Exacto, primero encriptas la información para esconder el contenido de tus mensajes (utilizando el **método de encriptación**). Luego, debes codificar el mensaje (utilizando el **método de codificación**) que será enviado entre cliente y servidor.
